index.js

```diff
@@ -47,7 +47,7 @@ mongoClient.connect(mongoUrl, {useNewUrlParser: true}, function(err, con) {
 
 
     // Manager
-    const manager = new Manager();
+    const manager = new Manager(db);
```

Comment:
Das passt.
Könntest du aber bitte zusätzlich die db variable mit const oder let deklarieren? (Zeile 21)
Sonst ist db weiterhin in mehreren Dateien als globale Variable verfügbar

Reply:
done :)